index.bs

```diff
@@ -283,8 +283,8 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
     [=authenticatorGetAssertion=] operation.
 
 : <dfn>Authenticator</dfn>
-:: A cryptographic device used by a [=[WAC]=] to (i) generate a [=public key credential=] and register it with a [=[RP]=],
-    and (ii) subsequently used to cryptographically sign and return, in the form of an [=Authentication Assertion=], a
+:: A cryptographic entity used by a [=[WAC]=] to (i) generate a [=public key credential=] and register it with a [=[RP]=],
+    and (ii) subsequently used to verify the user, cryptographically sign and return, in the form of an [=Authentication Assertion=], a
```

Comment:
This bit isn't really grammatical. How about a slightly bigger rewrite:

> (ii) [=authentication|authenticate=] by potentially [=user verification|verifying the user=], and then cryptographically    signing and returning, in the form of an [=Authentication Assertion=], a challenge and other data presented by a [=[RP]=]    (in concert with the [=[WAC]=]).

"potentially" because user verification isn't required.

Reply:
done.